src/main/java/com/commercetools/sync/customobjects/CustomObjectSyncOptionsBuilder.java

```diff
@@ -1,14 +1,15 @@
 package com.commercetools.sync.customobjects;
 
 import com.commercetools.sync.commons.BaseSyncOptionsBuilder;
+import com.fasterxml.jackson.databind.JsonNode;
 import io.sphere.sdk.client.SphereClient;
 import io.sphere.sdk.customobjects.CustomObject;
 import io.sphere.sdk.customobjects.CustomObjectDraft;
 
 import javax.annotation.Nonnull;
 
 public final class CustomObjectSyncOptionsBuilder extends BaseSyncOptionsBuilder<CustomObjectSyncOptionsBuilder,
-        CustomObjectSyncOptions, CustomObject, CustomObjectDraft> {
+        CustomObjectSyncOptions, CustomObject<JsonNode>, CustomObjectDraft<JsonNode>> {
```

Comment:
Will it also work for custom objects like this one? Especially check `hasIdenticalValue()`.

```
    {
      "container": "stock-full-import",
      "key": "allowed",
      "value": false
    }
```

Reply:
It can also be Number etc., so you should also test that. https://docs.commercetools.com/http-api-projects-custom-objects#representations

Reply:
JsonNode itself supports these types.
I added few more unit test case to verify it.
https://github.com/commercetools/commercetools-sync-java/pull/561/commits/f81ccd39df4c3351bf3ed448660737354070ae1f